scripts/setup-linux.sh

```diff
@@ -2,3 +2,12 @@
 sudo apt -y update
 sudo apt-get install -y protobuf-compiler libseccomp-dev
 
+echo "setting up rust"
+sudo rustup toolchain install 1.74 --component clippy --component rustfmt --no-self-update
```

Comment:
Doesn't rust-toolchain already enforce the right rust version? I am concerend about having to maintain rust versions in two difference places

https://github.com/spinkube/containerd-shim-spin/blob/main/rust-toolchain.toml

Reply:
oh, good to know. I will remove the explicit version from here.

Reply:
done